src/common/pf_lldp.h

```diff
@@ -26,6 +26,53 @@
 extern "C" {
 #endif
 
+#define PF_LLDP_SUBTYPE_MAC              4
+#define PF_LLDP_SUBTYPE_LOCALLY_ASSIGNED 7
+
+/**
+ * Check if information about peer device has been received
+ * 
+ * The info is received in an LLDP packet and relevant
+ * info is stored by LLDP stack.
+ * 
+ * @param net              In:    The p-net stack instance
+ * @param timestamp_10ms   Out:   Time when the LLDP packet with the info
+ *                                was first received, in units of
+ *                                10 milliseconds with system startup
+ *                                as the zero point (as in SNMP sysUpTime).
+ *                                Note that if an LLDP packet with identical
+ *                                info is received, the timestamp is not
+ *                                updated.
+ */
+bool pf_lldp_is_peer_info_received (
+   pnet_t * net, 
+   uint32_t * timestamp_10ms);
+
+/**
+ * Get Chassis ID of local device.
+ *
+ * @param net              In:    The p-net stack instance
+ * @param p_chassis_id     Out:   Chassis ID of local device.
+ */
+void pf_lldp_get_chassis_id (
+   pnet_t * net, 
+   pf_lldp_chassis_id_t * p_chassis_id);
+
+/**
+ * Get Chassis ID of peer device.
+ *
+ * The peer Chassis ID is received in LLDP frames. If no LLDP frame has been
+ * received, the returned Chassis ID is empty.
+ *
+ * Thread-safety is ensured by means of a mutex.
+ *
+ * @param net              In:    The p-net stack instance
+ * @param p_chassis_id     Out:   Chassis ID of peer device.
+ */
+void pf_lldp_get_peer_chassis_id (
```

Comment:
We need in a few months to support multiple ports (and thus multiple peers).
Would it be possible to use `pnet_lldp_peer_info_t` instead of` pnet_t` as arguments?
This also applies to `pf_lldp_is_peer_info_received()` if we somehow can store the info properly.

It is not entirely clear to me if this function is internal only, or should be used directly by the SNMP implementation. It affects how the arguments should look like.

Reply:
Use of pnet_t* as argument is needed for using the mutex. When multiple port support is added, the function will need to be modified, e.g. by adding a argument for the port number.

The function is not internal to the LLDP module. It will be used by the SNMP implementation.